workflow/type.go

```diff
@@ -15,8 +25,46 @@ type task struct {
 	TaskKey      string
 }
 
+type filter struct {
+	Key       string
+	Predicate predicate
+	Value     interface{}
```

Comment:
Unsued value, what for?

Reply:
`Value` is used in the `Match` function of the filter